templates/zerver/integrations/giphy.md

```diff
@@ -1,14 +1,15 @@
-# GIPHY GIF integration
+# Zulip GIPHY GIF integration
 
 Send animated GIFs with your message using GIPHY integration.
 
-![](/static/images/integrations/giphy/GIPHY_zulip.png)
+!!! tip ""
 
-GIPHY is enabled by default in Zulip Cloud. If the you're on a
-self-hosted server where the [GIPHY integration][help-center-giphy]
-isn't already configured, see the [configuration
-instructions][configure-giphy] to enable it.
+    GIPHY is enabled by default in Zulip Cloud. If you're on a
+    self-hosted server where the [GIPHY integration][help-center-giphy]
+    isn't already configured, see the [configuration
+    instructions][configure-giphy] to enable it.
```

Comment:
What if we did something like ...

```
1. Follow [these instructions](rtd-link) to configure this integration on a self-hosted Zulip server.

tip - GIPHY is enabled by default on Zulip Cloud

1. See our help center documentation on including [animated GIFs](help-link) in a message.
```

Not 100% sure that would look good and it'd likely be good to get Alya's feedback on that version.